packages/vuetify/src/components/VList/VListItem.sass

```diff
@@ -341,24 +341,35 @@
 
   @media (forced-colors: active)
     .v-list-item
-      &:not(&--active)
+      &--link:not(&--active)
         color: buttontext
 
+      &--link[href]:not(&--active)
+        color: unset
+
+      &--variant-text.v-list-item--active,
       &--variant-plain.v-list-item--active
         color: highlight !important
 
-      &:not(&--variant-plain).v-list-item--active
+        .v-list-item-title,
+        .v-list-item-subtitle,
+        .v-list-item-action
```

Comment:
I was looking at the example markup from the description with a small change to use `to` instead of `href` and have one of the links `/` (matching current page). The icon was still yellow instead of cyan. It can be fixed by adding `.v-icon` to this list.

BTW. Do you remember how I could further adjust the demo to verify those other classes are needed here? Just want to verify those are necessary, as they stand out a bit.